enterprise/views.py

```diff
@@ -960,6 +960,10 @@ def get(self, request, enterprise_uuid, course_id):
             lms_root_url,
             '/courses/{course_id}/courseware',
         )
+        LMS_START_PREMIUM_COURSE_FLOW_URL = urljoin(
+            get_configuration_value('LMS_ROOT_URL', settings.LMS_ROOT_URL),
```

Comment:
This variable is already defined. @Squirrel18 
```suggestion
            lms_root_url,
```